### ISSUE TITLE
[a11y]: The 'id' "ccs-:r2b:-tabpanel-0" specified for the ARIA property 'aria-controls' value is not valid

### ISSUE THREAD

**Issue by NabeelSiddiqui7** [carbon-design-system/carbon#18995]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

1.79.0

### React version

18.3.1

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker - Latest Deployment

### Description

Should pass IBM accessibility checker or at least be configurable

### WCAG 2.1 Violation

IBM 4.1.2 Name, Role, Value

### Reproduction/example

https://stackblitz.com/edit/github-ktlsb41q?file=src%2FApp.jsx

### Steps to reproduce

Run Accessibility Checker on Tabs

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by 2nikhiltom** [comment]
Hey @NabeelSiddiqui7 
The `TabPanel` components is missing in your usage. When you use the Tab component without a corresponding TabPanel, it creates an ARIA relationship using `aria-controls` that points to a non-existent element
The aria-controls attribute is referencing `ccs-«r1»-tabpanel-0`, but that element doesn't exist in your DOM 

below usage should fix the issue
```

let carbonTabs = (
    <Tabs>
      <TabList>
        <Tab key={'hello'} title={'Hello'}>
          Hello
        </Tab>
      </TabList>
      <TabPanels>
        <TabPanel>Tab Panel 1</TabPanel>
      </TabPanels>
    </Tabs>
  );
  return (
    <div>
      <Button>Hello world</Button>
      {carbonTabs}
    </div>
  )
```

**Comment by 2nikhiltom** [comment]
Hey, @NabeelSiddiqui7 
are you still blocked? could you confirm the fix so that we close this issue ?

**Comment by annawen1** [comment]
Closing issue as resolution has been provided above, feel free to reopen if the issue still persists.
